src/System/ref/System.csproj

```diff
@@ -0,0 +1,24 @@
+﻿<?xml version="1.0" encoding="utf-8"?>
```

Comment:
The more I think about it for consistency can you please move all these ref assemblies into a `ref\compat` folder. That will help make it more clear what these are for.